lib/phoenix/router.ex

```diff
@@ -420,8 +421,9 @@ defmodule Phoenix.Router do
     routes_with_exprs = Enum.map(routes, &{&1, Route.exprs(&1)})
 
     helpers_moduledoc = Module.get_attribute(env.module, :helpers_moduledoc)
+    trailing_slash = Module.get_attribute(env.module, :trailing_slash)
 
-    Helpers.define(env, routes_with_exprs, docs: helpers_moduledoc)
+    Helpers.define(env, routes_with_exprs, docs: helpers_moduledoc, trailing_slash: trailing_slash)
```

Comment:
It seems this option is not being used in `Helpers.define`. Once we decide which mode we want to support, then we can decide what to do with this. :)

Reply:
You're correct, this carried over from an earlier experiment.